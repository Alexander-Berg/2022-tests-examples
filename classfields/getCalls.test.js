jest.mock('www-cabinet/react/dataDomain/dashboard/helpers/calculateCallsChartData', () => {
    return () => 'CHART_DATA';
});

const { getBunkerMock } = require('autoru-frontend/mockData/state/bunker.mock');
const statsCallsDailyListMock = require('../mocks/withStatsCallsDailyList.mock');

const getCalls = require('./getCalls');

const state = {
    dashboard: {
        dateLimits: { from: '2019-10-11', to: '2019-10-14' },
        statsCallsDailyList: { calls_by_day: statsCallsDailyListMock },
    },
    bunker: getBunkerMock([ 'cabinet/dashboard' ]),
    config: { client: { salon: { poi: { call_tracking_on: '1' } } } },
};

it(`должен отдать правильную инфу о звонках`, () => {
    const result = getCalls(state);

    expect(result).toMatchSnapshot();
});
