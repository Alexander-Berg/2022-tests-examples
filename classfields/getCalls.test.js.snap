// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`должен отдать правильную инфу о звонках 1`] = `
Object {
  "bunkerDict": Object {
    "missing": "Пропущено",
    "moreButton": "Подробнее",
    "noCallTracking": Object {
      "addCallTracking": "Подключить",
      "text": "для отслеживания звонков подключите коллтрекинг. это бесплатно",
    },
    "noOffers": Object {
      "text": "После добавления объявлений здесь будет отображаться статистика принятых и пропущенных вызовов",
    },
    "noStats": Object {
      "text": "Звонков не поступало. Добавляйте больше фотографий, продвигайте объявления",
    },
    "received": "Принято",
    "title": "Звонки",
  },
  "chartData": "CHART_DATA",
  "hasCallTracking": true,
  "hasDataFromBackend": true,
  "hasStats": true,
  "missing": 7,
  "received": 9,
  "total": 16,
}
`;
